game/theater/start_generator.py

```diff
@@ -379,21 +356,20 @@ def generate_tgo_for_scenery(self, scenery: SceneryGroup) -> None:
             Heading.from_degrees(0),
             self.control_point,
         )
-        ground_group = GroundGroup(
+        ground_group = TheaterGroup(
             self.game.next_group_id(),
             scenery.zone_def.name,
             PointWithHeading.from_point(scenery.position, Heading.from_degrees(0)),
             [],
             g,
         )
-        ground_group.static_group = True
         g.groups.append(ground_group)
         # Each nested trigger zone is a target/building/unit for an objective.
         for zone in scenery.zones:
-            scenery_unit = SceneryGroundUnit(
+            scenery_unit = SceneryUnit(
                 zone.id,
                 zone.name,
-                "",
+                dcs.statics.Fortification.White_Flag,
```

Comment:
Ok i have to take care about the display_name of SceneryObjects as it will now show the White Flag :D 
![grafik](https://user-images.githubusercontent.com/6678803/153627657-a7d0519e-4410-463c-a1d4-5bd5b8981910.png)